templates/master/01-master-kubelet/_base/units/kubelet.yaml

```diff
@@ -22,6 +22,10 @@ contents: |
         --container-runtime=remote \
         --container-runtime-endpoint=/var/run/crio/crio.sock \
         --node-labels=node-role.kubernetes.io/master,node.openshift.io/os_id=${ID} \
+{{- if .KubeletIPv6}}
+        --node-ip :: \
+        --address :: \
```

Comment:
--address 0.0.0.0
--
  | The  IP address for the Kubelet to serve on (set to 0.0.0.0 for all IPv4  interfaces and `::` for all IPv6 interfaces) (default 0.0.0.0)  (DEPRECATED: This parameter should be set via the config file specified  by the Kubelet's --config flag. See  https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/  for more information.)

Reply:
Thanks for pointing out that this was deprecated.  I missed that.